app/assets/stylesheets/partials/_sections.scss

```diff
@@ -73,13 +73,37 @@
       }
     }
 
-    &.submit {
+    &.submit, &.analyse {
```

Comment:
We repeat the same colour twice here. I understand that this is to keep things consistent. But can we try one more variation - to use the second colour from the BIP logo? Maybe that one will suit nicely?